### ISSUE TITLE
Results rounded to 1 decimal place

### ISSUE THREAD

**Issue by adamh-broad** [Teichlab/cellphonedb#11]
Hi, thanks for a great package! I ran the code and the results look good, but all of the p-values are rounded to a single decimal point so its impossible to tell the difference between significant hits and noise. Is there a way to get the full p-values?

**Comment by mvento** [comment]
Hi @adamh-broad,

We are released the new Cellphonedb version v1.1.0. Now, you can set the result precision decimal using the parameter --result-precision=x (ie: result-precision=3).

Please, ask us if you have any other questions. 

Thanks for using Cellphonedb!

**Comment by hamishking** [comment]
Hi @mvento 

I've tried out the latest version (1.1.0) on the test datasets with default settings and changing the result-precision parameter (3, 5, 10 and 20). In my hands the result-precision only affects the rounding of the mean expression value reported, not the p values. Is this just a simple rounding error to correct?

For my actual dataset I have a very large number of the significant p values reported as zero. Apart from the precision of p value reporting (I tried up to result-precision=50), is there another reason that zeroes would be reported for the statistical analysis?

Thanks for your help with this, and a really nice method and package!
